### ISSUE TITLE
C-c prefix is not removed from input prefix keys

### ISSUE THREAD

**Issue by jtbm37** [ch11ng/exwm#390]
`M-x version`
>GNU Emacs 25.2.2 (x86_64-pc-linux-gnu, GTK+ Version 3.22.21) of 2017-09-22, modified by Debian

Consider the following
```
(defun my-exwm-finish-hook ()
  (when (and exwm-class-name
	     (or
	      (string= (downcase exwm-class-name) "virtualbox")
	      (string= (downcase exwm-class-name) "urxvt")))
    (setq-local exwm-input-prefix-keys '(?\C-i))))


(add-hook 'exwm-manage-finish-hook 'my-exwm-finish-hook)
```
When opening `urxvt`, `C-c` is still forwarded to emacs.

Expected: `C-c` is forwarded to `urxvt`.

Regression introduced by commit a6cfe3f22d4dff076e3e5da6676c411a52d6590e

Reverting the commit fixes the issue.

**Comment by rayslava** [comment]
Same with `C-d` and `C-a`

**Comment by ch11ng** [comment]
This behavior is somewhat changed.  The default <kbd>C-c</kbd> has to be disabled with

```elisp
(define-key exwm-mode-map (kbd "C-c") nil)
```

**Comment by jtbm37** [comment]
@ch11ng Are you sure ? That did not work for me. `C-c` is still passed to emacs.

**Comment by jtbm37** [comment]
ah gotta get the very latest ! fixed by dd0f10775a1893abe3b5c753e9db12d1397c2f6b

Thank you for your prompt support.
